testsuite/regressiontests/unused_names.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# unused_names.py
```

Comment:
I would suggest `test_unusued_names.py` as name and placing this in unittests, as the test is not directly motivated by a reported issue.

Reply:
Sounds reasonable. I've renamed/moved the file in 69456a5a.